draft-linker-digital-emblem.md

```diff
@@ -212,6 +230,23 @@ However, marking entire networks should drastically ease the burden of deploymen
 Notably, a digital emblem cannot be applied to infrastructure that is used for both services worthy and unworthy of protection.
 A digital emblem must always identify assets that only serve purposes that enjoy protection under IHL.
 
+# Proposed Designs for a Digital Emblem {#proposals}
+
+Two designs were proposed to solve the problem of a digital emblem.
+First, JHU APL proposed a DNS-based solution for a digital emblem {{?I-D.haberman-digital-emblem}}.
+Second, CECYT (a joint endeavour between ETH Zurich and Bonn University) proposed *An Authentic Digital EMblem (ADEM)*, which was initially described in an academic publication {{ADEM}}, but was also specified technically {{ADEM-SPEC}} and has openly accessible, working prototypes ([Go library and CLI](https://github.com/adem-wg/adem-proto), [Browser extension](https://github.com/adem-wg/adem-chrome)).
+We focus here on ADEM as it was selected by the ICRC as the most suitable proposal for a digital emblem.
```

Comment:
```suggestion
We focus here on ADEM as it was selected by the ICRC as the most suitable proposal for their digital emblem requirements. In terms of scope, this means meeting the need to apply an emblem to digital assets without a need to apply digital emblems to physical assets, which goes beyond the ICRC's needs.
```
I think something like this (doesn't have to be exactly this) would help put context around the decision and not just say "ADEM is the winner, moving on" which the original comes across as to a reader with zero background in the digital emblem space. It's not that JHU APL's solution can't solve other problems, but that ADEM focuses on the problem space the ICRC cares about.

Reply:
I think we should fight for the term "digital emblem" meaning the digital emblem as discussed in our draft. You might use other technology to solve other problems, but we should not call these other problems digital emblems. If you want to replace the physical emblems with something digital, you must go through the ICRC, and they fight this idea.

Happy to tone down what ADEM was selected for, but I think we should really insist on "'digital emblem' applied to physical assets" ≠ "digital emblem."

Reply:
Maybe we can go another way and rather stress that we understand "digital emblem" to only apply to digital assets and then list the reasons for the ICRC why they want that. This would also help to kickstart the discussion on this topic at the IETF. (And reduce surprises to ADs!)

Reply:
Ah, I see the nuance. Let me try again.